### ISSUE TITLE
Reproduce sklearn.metrics.classification_report

### ISSUE THREAD

**Issue by yujonglee** [repllabs/fastrepl#6]
There are some inconsistency exists though...

https://discuss.huggingface.co/t/help-model-evaluation-for-ner-yields-different-results-sklearn-vs-metric-compute/10528

https://github.com/huggingface/evaluate/blob/af3c30561d840b83e54fc5f7150ea58046d6af69/metrics/seqeval/seqeval.py

We added some tests in #5.
